### PR TITLE
Slack integration

### PR DESCRIPTION
**Pull request by toser**
wrote this some time ago. have phun :)

### CODE REVIEW

File: src/backend/witness/slack/slackbot.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      console.log(
+        `${watchers.length} Watcher${
+          watchers.length > 1 ? 's' : ''
+        } started in these channels: ${activeChats.map(x => x.name).join(', ')}`
+      )
+    })
+
+    bot.on('message', data => {
+      // all ingoing events https://api.slack.com/rtm
+      // console.log('new DATA: ------> ', data)
```
Comment (TUNER88) [inline src/backend/witness/slack/slackbot.js]: Could you remove it please?

**Review by TUNER88 — COMMENTED** [review]
Awesome contribution 👍

File: src/frontend/components/Slackbot/Log.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          onChange={e => {
+            setAutoScroll(e.currentTarget.checked)
+          }}
+        />{' '}
+        auto scroll
+      </label>
+      <pre className="slackbot__log" id="slackbot-log">
+        {log.map(({ stamp, type, message }) => (
+          <span key={stamp} className={`slackbot__log-line slackbot__log-line--${type}`}>
+            <span className="slackbot__log-stamp">{moment(stamp).format('HH:mm:ss')}]</span>{' '}
```
Comment (thbaumbach) [inline src/frontend/components/Slackbot/Log.js:21]: missing `[` before the timestamp

**Comment by thbaumbach** [comment]
Tested. Nice work 👍 

A small hint about the Slack feature within the README would be cool.

**Comment by Bastl34** [comment]
quick code review: LGTM

**Review by thbaumbach — CHANGES_REQUESTED** [review]

File: src/frontend/components/Slackbot/Log.js (hunk 2 above)
Comment (toser) [reply src/frontend/components/Slackbot/Log.js:21]: yep. it was intentional..

image

Comment (thbaumbach) [reply src/frontend/components/Slackbot/Log.js:21]: it looks weird imho

anyway, if the other guys are ok with the current state then LGTM
@Bastl34 @TUNER88

Comment (Bastl34) [reply src/frontend/components/Slackbot/Log.js:21]: I'm ok with this

**Review by Bastl34 — APPROVED** [review]

**Review by thbaumbach — APPROVED** [review]

**Comment by thbaumbach** [comment]
Thank you 👍

**Pull request merged**